charts/prometheus-thanos/templates/compact-statefulset.yaml

```diff
@@ -11,7 +12,7 @@ metadata:
 spec:
   replicas: 1
   updateStrategy:
-    type: {{ .Values.compact.updateStrategy }}
+    type: {{ $values.updateStrategy }}
```

Comment:
why is this changed? 
we like to configure this separately for all components.

Reply:
The functionality is not changed. Look at line 2 above: `$values` is the local variable I introduced for improved readability. It equals `.Values.compact`.

This also helps comparing templates across components (e.g. diff `compact-statefulset.yaml` and `store-gateway-statefulset.yaml`.

Reply:
OK, but we should keep this consistent in the whole chart.